### ISSUE TITLE
Date and Time error

### ISSUE THREAD

**Issue by davidmanoel** [ghostbsd/issues#49]
### Version

22.10.12

### Explain the issue and what happens

The problem/bug is happening when, in live mode or even after having the system installed, updated & upgraded or not, in versions 22.06 or 22.10, when I open the control center and acess date and time manager config, a dialog box with the name (GetNtpState ) show the error:

GDBus.Error:freedesktop.DBus.Error.ServiceUnknow: The name org.freedesktop.timedate1 was not provided by any .service files

When I press "OK", the configuration box does not allow me to enable NTP sync, showing Timezone (null) and not allowing me to make changes to these settings.

![GetNtpStateERROR](https://user-images.githubusercontent.com/113417875/197336983-bed2cad9-1ead-4cb7-af53-8c0e4db05f8b.png)

![Date TimeERROR](https://user-images.githubusercontent.com/113417875/197336990-44cc8fca-0400-4dd7-b8a3-c5bc963c6c9c.png)

### Explain how to reproduce the bug?

When opening the control center and acessing date and time manager config, the dialog box with the name (GetNtpState ) show the error.

### Additional context (if applicable)

This error is preventing me from successfully installing PORTS via terminal or Software Station.

When trying to install "ports" through the terminal, fetching, install and extracting happen perfectly from what I see, and then the following error occurs:

" pkg: Fail to set time on /usr/ports/security/p5-Crypt-NULL: No such file or directory"

I'm sure the 'ports' installation error is directly linked to the Date&Time configuration error in the Control Center.

I've reviewed the mo rc.conf settings and everything seems to be perfect.

Screenshots follow.

![ports install fail terminal](https://user-images.githubusercontent.com/113417875/197337928-a38200fa-d5ff-424a-a391-410a8dcfeb73.png)

![rcconf](https://user-images.githubusercontent.com/113417875/197337931-b0e2d6ae-66b3-4d2e-9774-3edee2e4eb2a.png)

**Comment by ericbsd** [comment]
issue moved from ghostbsd/ghostbsd-src to ghostbsd/issues

**Comment by brianthehughes** [comment]
@mfjurbala 

I ended up here after I noted that the Lock Screen shows time in UTC while the time in the desktop panels follows the location information provided during install. 

24.10.1 Mate 1.28.2

Almost 2 years ago... you suggested _We could write a guide for the TUI bsdconfig or possibly try to write a gui for it or just the NTP and timezone portion._

Perhaps an alternative is to finish stripping out the **Time and Date Manager (TDM)** and have GhostBSD utilize the **Time/Clock Preferences** component as the _General, Locations, Weather, Time Settings_ features all work there. I am taking a small leap assuming Time/Clock Preferences is not built on the TDM.

My opinion is that ntp/ntpd can be left to the advanced user as a command line experience.

 ![Image](https://github.com/user-attachments/assets/8cf52486-c678-4aed-8d05-b83a269b7a33)

**Comment by mfjurbala** [comment]
@brianthehughes I agree with your suggestion and it looks like TDM is already gone. I started looking into this but then had to step away (for longer than I anticipated). I tried to figure out the problem on and off and I just found the problem.

When you select your timezone in the installer the file /etc/localtime is created but /var/db/zoneinfo is not. When I would run bsdconfig or tzsetup that would generate /var/db/zoneinfo. So we just need to update the installer to have it create /var/db/zoneinfo and this issue should finally be fixed (I think).
